VERSION

```diff
@@ -0,0 +1 @@
+0.0.8
```

Comment:
Currently version comes from the git tag, there's no workflow to maintain this file and I don't think there's any need for it

Reply:
I don't like this pattern and I don't want to manage any version tracking files, git tags are the source of truth for that right now and I didn't hit any issues with that. `promu` seems very intrusive tool that creates a set of requirements and constrains that I don't feel entirely happy with. I think I prefer the `bakealite` approach